<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="COVID-19Data.db" uuid="45f2167c-c272-474c-826e-2d1966aebbde">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:$PROJECT_DIR$/main/COVID-19Data.db</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="COVID-19Data_delete" uuid="e69ae9c3-4d2e-4852-92a3-9cefcbc27609">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Code\Python\COVID-19_Data\delete\COVID-19Data_delete.db</jdbc-url>
      <working-dir>$ProjectFileDir$</working-dir>
      <libraries>
        <library>
          <url>file://$APPLICATION_CONFIG_DIR$/jdbc-drivers/Xerial SQLiteJDBC/3.34.0/sqlite-jdbc-3.34.0.jar</url>
        </library>
      </libraries>
    </data-source>
  </component>
</project>